back/engines/commercial/machine_translations/app/services/machine_translations/machine_translation_service.rb

```diff
@@ -4,6 +4,11 @@ module MachineTranslations
   class MachineTranslationService
     def build_translation_for(translatable:, attribute_name:, locale_to:)
       multiloc = translatable[attribute_name]
+      
+      # Occasional SQL-injection attacks have been attempted via values for attribute_name,
+      # so we return nil here. This also means we can avoid associated Sentry errors.
+      return nil if multiloc.nil?
```

Comment:
Returning `nil` means that, as well as being able to prevent the Sentry error, we ensure that any attempted SQL-injection value is lost at this point.

Reply:
Makes sense. Maybe you can also add this comment to the code.